### DISCUSSION TITLE
Is there a way to cancel the task according to the taskid without using the cli?

### DISCUSSION THREAD

**Discussion by hungtcs** [hibiken/asynq#695] [Q&A] [answered]
In some cases, I need to cancel the task across processes.
I saw that the cli has a command to cancel the task according to the taskid, but I did not find the API for the go code.
thanks.

**Comment by aaps3579** [comment, marked as answer]
Use the inspector client API provided to cancel the task.
`asynq.NewInspector(<connection_opts>).CancelProcessing("<your_task_id>")`

For more info on available methods https://github.com/hibiken/asynq/blob/master/inspector.go#L26
